contrib/cscope/packages.el

```diff
@@ -0,0 +1,60 @@
+;;; packages.el --- cscope Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2012-2014 Sylvain Benner
+;; Copyright (c) 2014-2015 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(setq cscope-packages '(evil-jumper
+                        helm-cscope
+                        xcscope))
+
+(setq cscope-excluded-packages '())
+
+(defun cscope/post-init-evil-jump ()
+  (defadvice helm-cscope-find-this-symbol (before cscope/goto activate)
+    (evil-jumper--push)))
+
+(defun cscope/init-helm-cscope ()
+  (use-package helm-cscope
+    :config
+    (use-package xcscope
+      :config
```

Comment:
Ok I see what @trishume meant by mashed-up thing :-)
This nested use-package should go in its own `cscope/init-xcscope`.